lldb/test/API/functionalities/data-formatter/swift-unsafe/main.swift

```diff
@@ -117,7 +117,43 @@ func main() {
     //%            '\[1\] = 0[xX][0-9a-fA-F]+ \(number_value = 3.14[0-9]*\)',
     //%            ])
   }
-  
+
+  // UnsafeRawBufferPointer
+  let bytes = [UInt8](0...255)
+
+  bytes.withUnsafeBufferPointer {
+    let buf = $0
+    let rawbuf = UnsafeRawBufferPointer(buf)
+    //% self.expect("frame variable -d run-target rawbuf",
+    //%            patterns=[
+    //%            '\(UnsafeRawBufferPointer\) rawbuf = 256 values \(0[xX][0-9a-fA-F]+\) {',
+    //%            '\[([0-9]+)\] = (\\1)'
+    //%            ])
+  }
+
+  // UnsafeMutableRawBufferPointer
+  var bits : [UInt8] = [0,1]
+
+  bits.withUnsafeMutableBufferPointer {
+    var mutbuf = $0
+
+    let mutrawbuf = UnsafeMutableRawBufferPointer(mutbuf)
+    //% self.expect("frame variable -d run-target mutrawbuf",
+    //%            patterns=[
+    //%            '\(UnsafeMutableRawBufferPointer\) mutrawbuf = 2 values \(0[xX][0-9a-fA-F]+\) {',
+    //%            '\[0\] = 0',
+    //%            '\[1\] = 1',
+    //%            ])
+
+    mutrawbuf.swapAt(0, 1)
+    //% self.expect("frame variable -d run-target mutrawbuf",
+    //%            patterns=[
+    //%            '\(UnsafeMutableRawBufferPointer\) mutrawbuf = 2 values \(0[xX][0-9a-fA-F]+\) {',
+    //%            '\[0\] = 1',
+    //%            '\[1\] = 0',
+    //%            ])
+  }
+
```

Comment:
Be good to test that "frame var mutrawbuf[0]" also works.

Reply:
I'll add a check for that